core/src/main/scala/kafka/tools/StorageTool.scala

```diff
@@ -170,7 +170,7 @@ object StorageTool extends Logging {
     printStream: PrintStream,
     validFeatures: java.util.List[Feature]
   ): Unit = {
-    val releaseVersion = Option(namespace.getString("release_version")).getOrElse(MetadataVersion.LATEST_PRODUCTION.toString)
+    val releaseVersion = Option(namespace.getString("release_version")).getOrElse(MetadataVersion.latestTesting().version())
```

Comment:
why to change the default value to `latestTesting`? it seems to me we should use production version by default